packages/react/src/combobox/root/ComboboxRoot.test.tsx

```diff
@@ -680,6 +680,57 @@ describe('<Combobox.Root />', () => {
       });
     });
 
+    it('loops ArrowDown from last to first by default', async () => {
```

Comment:
The test is duplicated, no? It's already included in the tests regarding the loopFocus prop.